packages/gatsby/src/utils/worker/render-html.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable @typescript-eslint/no-namespace, @typescript-eslint/naming-convention */
```

Comment:
Our lint rules want to discourage declaring namespace in source code files. Unfortunately having it in declaration file seemed to cause problems for packages other than gatsby ( https://app.circleci.com/pipelines/github/gatsbyjs/gatsby/64194/workflows/e70153e0-70d7-45b6-afa7-a7dcaf4a3471/jobs/707758 was failing for `gatsby-plugin-graphql-config`).

Naming convention disabled because of `Global` interface name not being prefixed with `I`